optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/SingleStatistic.java

```diff
@@ -178,6 +178,18 @@ public void readCsvStatisticFile() {
         }
     }
 
+    public void retrievePointList() {
+        if (getCsvFile().exists() && pointList == null) {
+            initPointList();
+            readCsvStatisticFile();
+        }
+    }
+
+    public void persistPointList() {
```

Comment:
the name "persist" in not entirely correct. It does more than persist them, it also removes them from memory.

What if we call these 2 methods "hibernatePointList" and "unhibernatePointList"? Although "hibernate" isn't perfect as it might make devs think of Hibernate ORM (the JPA impl). But I didn't find a good synonym.